wbooks/src/Books.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import myData from './books.json';
+import defaultBookIcon from './default_book.svg'
+import './Books.css';
+
+class Books extends Component {
+   constructor(props) {
+     super(props);
+     this.state = {
+       items: myData,
+     };
+     this.sortByField = this.sortByField.bind(this);
+   }
+
+   sortByField(props) {
+     var toFilter =  this.state.items;
+     var filter = props.filter;
+     for (var i = 0; i < toFilter.length -1; i++) {
```

Comment:
For statements are also rarely necessary.
Use array and string functions that are more declarative like:
arrays: `filter`, you also have: `map`, `find`. And for strings you can use `includes`.

Also you can access to object's attributes dynamically by using `[]`. for example: 
```js
const attr = 'title'; // or 'author'
console.log(book[attr]); // will print the books title.
```